featuretools/computational_backends/calculate_feature_matrix.py

```diff
@@ -167,25 +168,25 @@ def calculate_feature_matrix(features, entityset=None, cutoff_time=None, instanc
         pass_columns = [col for col in cutoff_time.columns if col not in ['instance_id', 'time']]
         # make sure dtype of instance_id in cutoff time
         # is same as column it references
-        target_entity = features[0].entity
-        dtype = entityset[target_entity.id].df[target_entity.index].dtype
-        cutoff_time["instance_id"] = cutoff_time["instance_id"].astype(dtype)
+        target_dataframe = features[0].dataframe
+        ltype = target_dataframe.ww.logical_types[target_dataframe.ww.index]
+        cutoff_time.ww.init(logical_types={'instance_id': ltype})
```

Comment:
will this potentially reset any existing woodwork typing if the user supplied a cutoff_time dataframe with woodwork already initialized?

Reply:
Yes - we could update the logic here to only change the type of the instance id column if WW is already initialized, otherwise we can init like this.

Reply:
@rwedge I initially thought this would be an easy fix, but it is not proving to be so simple. Do you think this is something we need to resolve for the initial release, or can we create an issue now and fix it later?